py3status/modules/weather_owm.py

```diff
@@ -0,0 +1,739 @@
+        This format accepts strftime/strptime placeholders to populate the
+        output with the time information.
+        Available placeholders:
+            icon
+        (default '{icon}: %X')
+    format_temperature: Formatting for temperature
+        Available placeholders:
+            current, icon, max, min
+        (default '{icon}: {current}°')
+    format_wind: Formatting for wind degree and speed
+        Available placeholders:
+            icon, degree, speed, gust
+        (default '{icon}: {speed} mph')
+    icon_atmosphere: Icon for atmospheric conditions, like fog, smog, etc.
+        (default '🌫')
+    icon_breeze: Icon for wind or breeze
+        (default '☴')
+    icon_cloud: Icon for clouds
+        (default '☁')
+    icon_extreme: Icon for extreme weather
+        (default '⚠')
+    icon_humidity: Icon for humidity
+        (default '●')
+    icon_pressure: Icon for pressure
+        (default '◌')
+    icon_rain: Icon for rain
+        (default '🌧')
+    icon_snow: Icon for snow
+        (default '❄')
+    icon_sun: Icon for sunshine
+        (default '☼')
+    icon_temperature: Icon for temperature
+        (default '○')
+    icon_thunderstorm: Icon for thunderstorms
+        (default '⛈')
+    icons: A dictionary relating weather code to icon
+        See https://openweathermap.org/weather-conditions for a complete list
+        of supported icons. This will fall-back to the listed icon if there is
+        no specific icon present.
+        There are multiple ways to specify individual icons based on the id:
+            - Use the key 'i601' to reference the condition with id = 601
+              (snow)
+            - Use the key 'i230_i232' to reference a span of conditions
+              inclusive, in this case conditions (230, 231, 232) (thunderstorm
+              with drizzle)
+        Also, you can specify the icons for the various formatting sections
+        below. For example, to specify the icon for 'format_pressure', use
+        'pressure'. A few formatting sections will take from the defaults or be
+        dynamic, including
+            - clouds
+            - rain
+            - wind
+            - sunrise
+            - sunset
+        These may be specified regardless.
+        (default None)
+    lang: An ISO 639-1 code for your language (two letters)
+        (default 'en')
+    location: A tuple of floats describing the desired weather location
+        The tuple should follow the form (latitude, longitude), and if set,
+        implicitly disables the IP-API Geolocation API.
+        (default None)
+    rain_unit: Unit for rain fall
+        Options:
+            cm, ft, in, mm, m, yrd
+        (default 'in')
+    request_timeout: The timeout in seconds for contacting the IP-API.
+        (default 10)
+    snow_unit: Unit for snow fall
+        Options:
+            cm, ft, in, mm, m, yrd
+        (default 'in')
+    temp_color: Color the temperature output based on a color scale.
```

Comment:
I would like to leave the default in for `thresholds`, since it gives the user a reasonable temperature color scale built in.

I do like this idea, but I'm not sure how to leave a good default for the user and eliminate the `temp_color` option. 

It could be accomplished by special casing on the `thresholds` value, by having hard-coded, recognized options such as `thresholds = 'basic'`, and handling it from there, but that seems clunky.

I'm open to suggestions!